libc/src/__support/libc_assert.h

```diff
@@ -71,7 +71,7 @@ LIBC_INLINE void report_assertion_failure(const char *assertion,
 
 #define LIBC_ASSERT(COND)                                                      \
   do {                                                                         \
-    if (!(COND)) {                                                             \
+    if (LIBC_UNLIKELY(!(COND))) {                                              \
```

Comment:
Can you also include `src/__support/macros/optimization.h` where these macros are defined and update cmake dependency?  Same for `libc/src/assert/assert.h`